### DISCUSSION TITLE
Rename `column()` table filter method in v3?

### DISCUSSION THREAD

**Discussion by titantwentyone** [filamentphp/filament#3998] [Ideas]
Would be a breaking change for v3 but a rename from `column` to something else (`field` perhaps?) may be warranted? `column` implies you should have a table column in order to use the method but any field on the model can be used.

**Comment by danharrin** [comment]
Sure! We can just make an alias for this column and it won't be a breaking change

**Comment by sohaibafifi** [comment]
What about using `attribute`?

**Reply by danharrin** [reply]
Thats even better
